content/applications/finance/accounting/receivables/customer_payments/followup.rst

```diff
@@ -14,31 +14,42 @@ Configuration
 =============
 
 To configure a :guilabel:`Follow-Up Action`, go to :menuselection:`Accounting --> Configuration -->
-Follow-up Levels`. Several follow-up actions are available by default, and you can customize them
-any way you want or create new ones to trigger the following actions:
+Follow-up Levels`. Several follow-up actions are available by default, but you can
+:guilabel:`Create` new ones if required. You can change its **name** and the **number of days**
+after which the follow-up actions are triggered. The follow-up :guilabel:`Actions` available are:
```

Comment:
- no need for that guilabel since you're not referring to an UI element on the screen. You're just saying that it can be created.
- "new ones" = plural, so the next sentence is wrong.
```suggestion
create new ones if required, and change their names and the number of days
after which the follow-up actions are triggered. The follow-up :guilabel:`Actions` available are:
```